AirCasting/Dashboard/DashboardView.swift

```diff
@@ -85,6 +85,13 @@ struct DashboardView: View {
         .onChange(of: selectedSection.section) { newValue in
             try! coreDataHook.setup(selectedSection: newValue)
         }
+        .onChange(of: selectedSection.mobileActiveSessionWasFinished) { newValue in
+            if newValue &&
+                sessions.count == 1 {
```

Comment:
Can it be a one-liner? I think it is easier to read when its size allows us to do that